ios/CredentialsManagerBridge.swift

```diff
@@ -82,9 +82,42 @@ public class CredentialsManagerBridge: NSObject {
         resolve(credentialsManager.clear())
     }
     
-    @objc public func enableLocalAuthentication(title: String?, cancelTitle: String?, fallbackTitle: String?) {
+    @objc public func enableLocalAuthentication(title: String?, cancelTitle: String?, fallbackTitle: String?, evaluationPolicy: NSNumber?) {
         let titleValue = title ?? "Please authenticate to continue"
-        self.credentialsManager.enableBiometrics(withTitle: titleValue, cancelTitle: cancelTitle, fallbackTitle: fallbackTitle)
+        let policyValue = self.convertPolicyInt(policyInt: evaluationPolicy);
+        self.credentialsManager.enableBiometrics(withTitle: titleValue, cancelTitle: cancelTitle, fallbackTitle: fallbackTitle, evaluationPolicy: policyValue as LAPolicy)
```

Comment:
`as LAPolicy` shouldn't be necessary, as `convertPolicyInt(policyInt:)` already returns a `LAPolicy`.